src/interface/emacs/khoj.el

```diff
@@ -634,7 +634,7 @@ CONFIG is json obtained from Khoj config API."
       (buffer-string)))
   ;; Update index on khoj server after configuration update
   (let ((khoj--server-ready? nil))
-    (url-retrieve (format "%s/api/update?t=org" khoj-server-url) #'identity)))
+    (url-retrieve (format "%s/api/update?t=org&client=emacs" khoj-server-url) #'identity)))
```

Comment:
Don't know enough e-lisp to tell you how you can do this, but would it make sense to have some string where you can format the values that are changing for each function? For example, in the Python way, I'd have a `url` variable as below and swap out parameters as needed.
```
url = f"/api/{function}?t=org&client=emacs"
```

Same suggestion for other endpoints for simplicity.

Reply:
I'm not sure how much reuse we get from doing this. as the api and request parameters are different across the different api endpoints. Let me know if it's fine to defer looking into this deeper a little later?